### DISCUSSION TITLE
ARM64 build of latest Net6 Having inconsistent behavior

### DISCUSSION THREAD

**Discussion by bstrother1** [unoplatform/uno#8236] [Q&A]
Greetings,
     So while testing the latest Uno Net6 Template on Mac with an ARM64 architecture I have been experiencing very inconsistent behavior that leads me to believe that the template currently doesn't support Mac Arm64. With a current out-of-box build of:

SDKS:
     6.0.100
     6.0.200
Visual Studio Preview 7.0.0.703 
Xcode-beta 13.3

Uno-check installed though that required me to change the DOTNET_ROOT to target the x64 folder version of Net 6.0.200 and updating the .csproj in mobile from osx-x64 to osx-arm64. An error persists intermittently:

/usr/local/share/dotnet/packs/Microsoft.macOS.Sdk/12.1.301-preview.13.7/targets/Xamarin.Shared.Sdk.targets(935,3): error: The AOT compiler '' does not exist.

Attached is a sample project that has now consistently shown this problem when both running dotnet build and when running dotnet build MyUnoApp.Mobile -t:Run -f net6.0-macos. 
[MyUnoApp.zip](https://github.com/unoplatform/uno/files/8190035/MyUnoApp.zip)

Let me know if I missed something. Thanks!

**Comment by jeromelaban** [comment]
Thanks for the report. This is an error that related to the .NET Runtime and tooling, which is likely to be fixed by the .NET team in upcoming releases. If the issue persists after preview 14, we can create an issue there: https://github.com/xamarin/xamarin-macios
